cmake/mirrors/gitcode.cmake

```diff
@@ -34,7 +34,11 @@ macro(ocv_download_url_gitcode_usercontent OWNER)
   string(REPLACE "/" ";" DL_URL_split ${DL_URL})
   list(GET DL_URL_split 5 __COMMIT_ID)
   list(GET DL_URL_split 6 __PKG_NAME)
-  set(DL_URL "https://gitcode.net/${OWNER}/opencv_3rdparty/-/raw/${__COMMIT_ID}/${__PKG_NAME}/")
+  if(__PKG_NAME)
+    set(DL_URL "https://gitcode.net/${OWNER}/opencv_3rdparty/-/raw/${__COMMIT_ID}/${__PKG_NAME}/")
```

Comment:
I propose something like this to not duplicate URLs and logic:
```
set(DL_URL "https://gitcode.net/${OWNER}/opencv_3rdparty/-/raw/${__COMMIT_ID}/")
if(__PKG_NAME)
  set(DL_URL "${DL_URL}${__PKG_NAME}/")
endif()
```

Reply:
Thanks for the suggestion! Code has been updated to your suggestion.